bridge/client/app/_components/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.ts

```diff
@@ -70,9 +94,19 @@ export class KtbProjectSettingsGitExtendedComponent {
       });
   }
 
+  private isRemoteUrlEmpty(): boolean {
+    if (!this.gitInputData) {
```

Comment:
In this case, we have a duplicated check for `!this.gitInputData`. This condition is checked before.
Option 1: remove condition in `ngOnInit`
```ts
public ngOnInit(): void {
    if (this.isRemoteUrlEmpty()) {
      ...
    }
}
```

Option 2: provide the data via parameter
```ts
public ngOnInit(): void {
    if (!this.gitInputData || this.isRemoteUrlEmpty(this.gitInputData)) {
      ...
    }
}
...
private isRemoteUrlEmpty(gitInputData: IGitDataExtended): boolean {
    return (
      (isGitHTTPS(gitInputData) && !gitInputData.https.gitRemoteURL) ||
      (isGitSSH(gitInputData) && !gitInputData.ssh.gitRemoteURL)
    );
  }
```

Reply:
Option 2, as I wanted it to be an explicit check in ngOnInit